src/dodal/devices/focusing_mirror.py

```diff
@@ -107,30 +114,49 @@ def voltage_channels(self) -> list[MirrorVoltageDevice]:
         ]
 
 
-class FocusingMirror(Device):
+class FocusingMirror(StandardReadable):
     """Focusing Mirror"""
 
-    def __init__(self, bragg_to_lat_lut_path, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+    def __init__(
+        self, name, prefix, bragg_to_lat_lut_path=None, x_y_sufffixes=("X", "Y")
```

Comment:
I wonder if a signature like 

```
self, name, prefix, *, bragg_to_lat_lut_path=None, x_suffix="X", y_suffix="Y"
```

would be a bit clearer? I don't love that we have to do `x_y_sufffixes[1]` later.